Fix minor issues from Hugo (#39)
chris-wood/draft-irtf-cfrg-voprf: draft-sullivan-cfrg-oprf.md

# OPRF Protocol {#protocol}

> In this section we describe the OPRF protocol. Let GG be a prime-order additive

Reviewer: GG is of prime order p, right? We should say so if that's the case.

~~~
Input:

>  l: Some suitable choice of key-length (e.g. as described in [NIST]).

Reviewer: Did {{NIST}} not work?

Author: No, because it's in a ~~~ block I believe

Steps:

>  1. N := (r^(-1)Z

Reviewer: ```suggestion
 1. N := (r^(-1))Z
```

Steps:

>  1. N := (r^(-1)Z

Reviewer: ```suggestion
 1. N := (r^(-1))Z
```

A valid way of computing additive blinding would be to instead compute
H_1(x)+rG, where G is the common generator for the group.

> By pre-processing tables of blinded scalar multiplications for the specific

Reviewer: I wonder if this is a choice we should specify. It seems use of a different blinding technique is an entirely separate protocol. Maybe we ought to (in a future issue) make the blinding choice a parameter and put it behind a parameterized function, e.g., `Blind()`?

Author: I agree, I think this would be a good change to make for a future version.

Author: I'll make a separate issue to track it after the submission today

 5. Output D := (c, s)
~~~

We note here that it is essential that a different r value is used for every
invocation. If this is not done, then this may leak the key k in a similar

> fashion as is possible in Schnorr/(EC)DSA scenarios where frash randomness is

Reviewer: ```suggestion
fashion as is possible in Schnorr or (EC)DSA scenarios where fresh randomness is
```
